beater/jaeger/grpc.go

```diff
@@ -154,3 +155,5 @@ func (s *grpcSampler) validateKibanaClient(ctx context.Context) error {
 	}
 	return nil
 }
+
+func boolean(b bool) *bool { return &b }
```

Comment:
```suggestion
func newBool(b bool) *bool { return &b }
```

This is somewhat personal taste, but I find it helpful to use `new<BasicType>` to indicate that we're creating a pointer to a value of the specified type, analogous to `new(SomeStructType)`.